src/scenes/PosterPage/PosterPage.tsx

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components';
+import Page from 'components/core/Page/Page';
+import { BaseM, BaseXL, TitleM } from 'components/core/Text/Text';
+import InteractiveLink from 'components/core/InteractiveLink/InteractiveLink';
+import Button from 'components/core/Button/Button';
+import { contentSize, pageGutter } from 'components/core/breakpoints';
+import colors from 'components/core/colors';
+import PosterFigmaFrame from './PosterFigmaFrame';
+import ActionText from 'components/core/ActionText/ActionText';
+import StyledBackLink from 'components/NavbarBackLink/NavbarBackLink';
+import { useIsMobileWindowWidth } from 'hooks/useWindowSize';
+import { useToastActions } from 'contexts/toast/ToastContext';
+import useTimer from 'hooks/useTimer';
+
+export default function PosterPage() {
+  const isMobile = useIsMobileWindowWidth();
+  const { pushToast } = useToastActions();
+
+  const FIGMA_URL = 'https://www.figma.com/file/YrjZkebwPMIPr3aFx8o2w1/Untitled?node-id=1%3A2';
```

Comment:
that's the difference between this URL and the figmaEmbedUrl?

Reply:
A little bit different. In the embed Figma, there is `https://www.figma.com/embed?embed_host=share&url=${FIGMA_URL}` in the front. 


I created a separate component for Embed Figma `<PosterFigmaFrame>` that accepts the `FIGMA_URL` as a prop. So, we can only manage it in one place.